cpu/cortexm_common/include/cpu.h

```diff
@@ -107,6 +107,14 @@ static inline void cortexm_sleep(int deep)
     unsigned state = irq_disable();
     __DSB();
     __WFI();
+    /*
```

Comment:
Let's move the comment into the ifdef, and make it a little shorter. E.g.:
```
/* STM32L152RE crashes without this __NOP(). See #8518. */
```

Reply:
Addressed.